src/allotropy/parsers/novabio_flex2/novabio_flex2_structure.py

```diff
@@ -116,11 +120,9 @@ def _get_properties(
         cls, data: SeriesData, property_mappings: dict[str, Any]
```

Comment:
It seems like this method is no longer necesary.

Reply:
Nice catch! Done!